paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -104,11 +105,13 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
     }
 
     @Test
-    fun `state has manage_all saved payment method action when multiple saved PMs are available`() {
+    fun `state has manage_all saved payment method action when multiple PMs are available, can edit and remove`() {
         runScenario(
             initialPaymentMethods = PaymentMethodFixtures.createCards(3),
-            allowsRemovalOfLastSavedPaymentMethod = false,
         ) {
+            canRemove.value = true
+            canEdit.value = true
```

Comment:
I think it'd be more idiomatic to include `canRemove` and `canEdit` as params of `runScenario`, rather than setting the flow's value here. We don't really need to expose the flows in the `TestParams` either, since these values never change during a test

Reply:
though -- should we have tests for canRemove or canEdit changing? (I'm not sure, wondering your opinion)

Reply:
Testing to see if the state reacts to changes in `canEdit` and `canRemove` might be good tests.